### DISCUSSION TITLE
Disable root app layout for certain route

### DISCUSSION THREAD

**Discussion by Jarco-dev** [vercel/next.js#42257] [Help]
I am yet to find it and it's a big yes or no as to whether I'll try out the app directory.

Is there a way to have a layout at the root level but disable that specific layout or overwrite it for let's say `/dashboard/*` as example?

**Comment by icyJoseph** [comment]
Refer to my answer here, I did a bit of docs reading and testing and managed to get it working:

https://github.com/vercel/next.js/discussions/42227

## Edit

Reposting for visibility and internet points. 

The secret sauce are groups!

```
❯ tree app/
app/
├── (everyone)
│   ├── layout.tsx
│   ├── login
│   │   └── page.tsx
│   └── page.tsx
└── (users)
    ├── dashboard
    │   └── page.tsx
    └── layout.tsx
```

Or something like

```
❯ tree app/
app/
├── (detailed)
│   ├── layout.tsx
│   └── orders
│       └── [id]
│           └── invoice
│               └── page.tsx
└── (root)
    ├── layout.tsx
    ├── orders
    │   └── [id]
    │       └── page.tsx
    └── page.tsx
```

In this case `app/orders/[id]/invoice` has a layout, defined right under `detailed`, meanwhile, `app/orders/[id]` has a layout defined right under `root`.
